src/Centreon/Domain/Monitoring/ResourceFilter.php

```diff
@@ -246,6 +251,25 @@ public function setHostgroupIds(array $hostgroupIds): self
         return $this;
     }
 
+    /**
+     * @return int[]
+     */
+    public function getMonitoringserverIds(): array
+    {
+        return $this->monitoringserverIds;
+    }
+
+    /**
+     * @param int[] $monitoringserverIds
+     * @return \Centreon\Domain\Monitoring\ResourceFilter
+     */
+    public function setMonitoringserverIds(array $monitoringserverIds): self
```

Comment:
```suggestion
    public function setMonitoringServerIds(array $monitoringServerIds): self
```
don't forget to update the method name when you call it (e.g ResourceRepository)